### PR TITLE
f-takeawaypay-activation@0.3.0 - Extracted styles to the separate file

### PR DESCRIPTION
**Pull request by oleksiishevchukje**
Code cleanup

### CODE REVIEW

**Review by ashleynolan — DISMISSED** [review]

File: packages/components/organisms/f-takeawaypay-activation/src/components/ActivationSuccessful.vue [hunk 1]
```diff
@@ -46,19 +46,6 @@ export default {
...
 
-.c-takeawaypayActivation-icon {
-    transform: translate(10%);
-    width: 125px;
-}
-
-.c-takeawaypayActivation-title {
-    @include font-size(body-l);
-    font-weight: $font-weight-bold;
-    padding-top: spacing(x2);
```
Comment (ashleynolan) [inline packages/components/organisms/f-takeawaypay-activation/src/components/ActivationSuccessful.vue:57]: Not sure if you need to add a line to cover this for this component specifically?

Comment (oleksiishevchukje) [reply packages/components/organisms/f-takeawaypay-activation/src/components/ActivationSuccessful.vue:57]: @ashleynolan Sorry, I didn't get it. Could you please rephrase? 🙂

Comment (oleksiishevchukje) [reply packages/components/organisms/f-takeawaypay-activation/src/components/ActivationSuccessful.vue:57]: Thanks @ashleynolan. It was really needed for all components, so added it back.

File: packages/components/organisms/f-takeawaypay-activation/src/components/ActivationLoggedIn.vue [hunk 2]
```diff
@@ -73,22 +73,6 @@ export default {
...
-}
-
-.c-takeawaypayActivation-email {
-    font-weight: $font-weight-bold;
-}
-
-.c-takeawaypayActivation-actions {
-    padding-top: spacing(x3);
-}
+@import './../assets/scss/takeaway-pay-activation';
```
Comment (matthewhardern) [inline packages/components/organisms/f-takeawaypay-activation/src/components/ActivationLoggedIn.vue:76]: Why are we moving these styles out of the component file?

Comment (oleksiishevchukje) [reply packages/components/organisms/f-takeawaypay-activation/src/components/ActivationLoggedIn.vue:76]: @matthewhardern They were duplicated between (sub)components and there was a suggested to extract them https://github.com/justeat/fozzie-components/pull/1078#discussion_r660828790

Comment (matthewhardern) [reply packages/components/organisms/f-takeawaypay-activation/src/components/ActivationLoggedIn.vue:76]: makes sense

**Review by ashleynolan — APPROVED** [review]

**Review by matthewhardern — APPROVED** [review]

**Pull request merged**